spring-social-microsoft-graph/src/test/resources/user.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "16f5a7b6-5a15-4568-aa5a-31bb117e9967",
+  "businessPhones": [
+    "123"
+  ],
+  "displayName": "Anne Weiler",
+  "givenName": "Anne",
+  "jobTitle": "Manufacturing Lead",
+  "mail": "annew@CIE493742.onmicrosoft.com",
+  "mobilePhone": "+1 3528700812",
```

Comment:
I think it's fictitious already. The data is from the Microsoft Graph Explorer. 

Click the `GET my profile` in the `Getting Started` tab, without logging in. 
https://developer.microsoft.com/en-us/graph/graph-explorer#